### PR TITLE
Update Tabs on Feature details & Experiment details pages

### PR DESCRIPTION
**Pull request by ahdriel**
### Features and Changes

Updated the tabs on Features & Experiments detail pages to use Radix components.

### Changelog

- Added new size `3` for TabsList (Radix supports only 1 and 2)
- Removed header white background color to match the new design
- Fixed the logic for the pinned header for Experiment details page

### Screenshots

| Before | After |
|--------|--------|
| features-before | features-after |
| experiments-before | experiments-after |

### CODE REVIEW

**Comment by ahdriel** [comment]
Ohh good catch on the shadow showing up when an Experiment is published -- it is a bug when we calculate if the Tabs are pinned or not.

You can see the desired effect on your Loom around the :22 mark where the tabs become pinned to the top of the page (we add a shadow when that happens to create better separation).

I'll take a look into the `Start Experiment` not recalculating the pinned property appropriately.

File: packages/front-end/styles/global-radix-overrides.scss [hunk 1]
```diff
@@ -270,3 +270,15 @@
...
+.rt-BaseTabList:where(.rt-r-size-3) {
+  font-size: var(--font-size-3);
+  line-height: var(--line-height-2);
+  letter-spacing: var(--letter-spacing-3);
+  --tab-height: var(--space-7);
+  --tab-padding-x: var(--space-2);
+  --tab-inner-padding-x: var(--space-2);
+  --tab-inner-padding-y: var(--space-1);
+  --tab-inner-border-radius: var(--radius-2);
+}
```
[the remark this answers is not in the document]
Comment (ahdriel) [reply packages/front-end/styles/global-radix-overrides.scss:284]: Oh yeah -- I copied from the DevTools by inspecting the element and seeing which variables were being used.

I don't think Radix has a list of all css variables that can be overridden unfortunately.

File: packages/front-end/components/Radix/Tabs.tsx [hunk 2]
```diff
@@ -64,11 +65,31 @@ export function Tabs({
...
 export function TabsList({
   children,
+  size = "2",
+  className,
   ...props
-}: React.ComponentProps<typeof RadixTabs.List>) {
-  return <RadixTabs.List {...props}>{children}</RadixTabs.List>;
+}: TabsListProps) {
+  const sizeValue = size === "3" ? "2" : size;
+  const classNameValue = size === "3" ? "rt-r-size-3" : "";
```
[the remark this answers is not in the document]
Comment (ahdriel) [reply packages/front-end/components/Radix/Tabs.tsx:82]: The `rt` prefix are for classes that are created by Radix. So usually we will only add new rules for `.rt-` rules when we want to override default behaviors.

In this case I used `rt-` to follow the same pattern as they are applying, but maybe I should actually remove it and make it clear that this is a custom implementation and not something that Radix provides out of the box.

I think `rt` stands for `Radix Themes`.

File: packages/front-end/styles/global-radix-overrides.scss (hunk 1 above)
[the remark this answers is not in the document]
Comment (mknowlton89) [reply packages/front-end/styles/global-radix-overrides.scss:284]: That's a good approach. Thanks!

File: packages/front-end/components/Radix/Tabs.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (mknowlton89) [reply packages/front-end/components/Radix/Tabs.tsx:82]: Thanks for explaining!

**Comment by ahdriel** [comment]
@mknowlton89 I fixed the bug -- the root cause is that when we are hiding the tabs I was defaulting to `0` as a fallback value. This meant that when the tabs were not being rendered, we considered that they were pinned, and when we displayed them (when the test is running) the page would not scroll and the state would persist.

The fix I decided on is to not modify the pinned property if the Tabs are not being rendered.

**Pull request merged**